### PR TITLE
Module 3. Components

### PR DESCRIPTION
**Pull request by VikaScherban**
- Add MovieTile, MovieDetails, SortControl components
- Add Storybook library and cover all components with stories
- Cover all components with unit tests

### CODE REVIEW

File: src/components/content/Content.tsx [hunk 1]
```diff
@@ -1,26 +1,45 @@
...
     }
 
+    const sortChanged = (newValue: SortByOption) => {
+        updateSorting(newValue);
+
+        console.log('Content, sortChanged', newValue);
+    }
+
+    const onMovieSelected = (movieId: number) => {
+        movieSelected(movieId);
```
Comment (DmitriyDubravin) [inline src/components/content/Content.tsx]: Namings should be easy to understand. Suggestion: selectMovieId

File: src/components/content/filter-line/genre-select/GenreSelect.tsx [hunk 2]
```diff
@@ -13,9 +13,6 @@ function GenreSelect({genres, currentGenre, genreSelected}: GenreData) {
                     >{genre.name}</li>
```
Comment (DmitriyDubravin) [inline src/components/content/filter-line/genre-select/GenreSelect.tsx:13]: You should add a unique 'key' to the 'li's

File: src/components/content/filter-line/sort-control/SortControl.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import {SortControlData} from "../../../../models/sortControl";
+
+function SortControl({currentSorting, sortChanged}: SortControlData) {
+    const onSortChange = (event: any)=> {
+        sortChanged(event.target.value);
+    }
+
+    return (
+        <div className="sort-by-block" data-testid="sort-control">
+            <button className="sort-by-button">SORT BY</button>
```
Comment (DmitriyDubravin) [inline src/components/content/filter-line/sort-control/SortControl.tsx]: Is it button?

File: src/components/content/filter-line/genre-select/GenreSelect.tsx (hunk 2 above)
Comment (VikaScherban) [reply src/components/content/filter-line/genre-select/GenreSelect.tsx:13]: Sorry, but I see it in the parameter `key={genre.id}`, please check it again

File: src/components/content/Content.tsx (hunk 1 above)
Comment (VikaScherban) [reply src/components/content/Content.tsx]: Thank you, renamed

File: src/components/content/filter-line/sort-control/SortControl.tsx (hunk 3 above)
Comment (VikaScherban) [reply src/components/content/filter-line/sort-control/SortControl.tsx]: Thank you, changed

**Pull request merged**